api/SVGViewElement.json

```diff
@@ -47,62 +47,6 @@
           "standard_track": true,
           "deprecated": false
         }
-      },
-      "viewTarget": {
-        "__compat": {
-          "support": {
-            "chrome": {
-              "version_added": true,
-              "version_removed": "56"
-            },
-            "chrome_android": {
-              "version_added": true,
-              "version_removed": "56"
-            },
-            "edge": {
-              "version_added": "12",
-              "version_removed": "79"
```

Comment:
This is unfortunately less than two years ago.

Reply:
@vinyldarkscratch are you going to repurpose this PR to some other end or should we close it?

Reply:
Right, legacy Edge still has a half a year to go...  Unfortunately I don't see a way to repurpose this PR, so unless we just want to skip ahead and remove it early, we'll have to reopen this PR another time.

Reply:
Or actually, maybe a single hardcoded date would do it, then we could move it forward as much as the data allows at any time.

Reply:
I was thinking about adding a linter, and I think I still have the code around.  The linter is based upon the current date, which can of course cause issues.  A hard-coded date makes much more sense!